main.py

```diff
@@ -1,25 +1,90 @@
-from loader import load_csv
-from merge import merge_audio_video
 import argparse
+import logging
+from datetime import datetime
+from os import mkdir
+from os.path import dirname, join, isdir, isfile
 
+from loader import DateBasedOrganizer, get_record_filenames
+from merge import move_files, copy_files
+from util import load_config, inject_config_if_missing
 
-def run(video_files_path):
-    combinations = load_csv()
-    merge_audio_video(combinations, video_files_path)
+# Setup logging
```

Comment:
mozno presunut do utils ako `setup_logger` ?

Reply:
Moze byt. Bude to cistejsie.